docs/audit.md

```diff
@@ -140,3 +140,126 @@ There are times you want to extract the raw secret values to run further analysi
 so with the `--raw` flag.
 
 TODO: Example when this feature is written up.
+
+## Report generation
```

Comment:
The audit documentation has been updated to include the new reporting feature information. It also includes some examples.